isofit/inversion/inverse_simple.py

```diff
@@ -309,18 +299,29 @@ def invert_analytical(
     xa_surface = xa_full[fm.idx_surface]
 
     if fm.RT.glint_model:
-        prprod = Sa_inv @ xa_surface
+        winglintidx = np.concatenate(
+            (winidx, fm.idx_surface[-2:]), axis=0
+        )  # Include glint indices
```

Comment:
We need to do this without hard-coding the positions.

Reply:
Good catch! I actually added the `idx_surf_nonrfl` attribute to the `ForwardModel` class some time ago, which was intended for exactly these cases.